src/main/java/Deadline.java

```diff
@@ -0,0 +1,49 @@
+import java.time.LocalDate;
+
+/**
+ * Represents a deadline task
+ */
+public class Deadline extends Task{
+    LocalDate deadline;
+    String time;
+
+    /**
+     * Constructor of a deadline
+     * @param name Name of the deadline
```

Comment:
It will be better if there is an empty line between description and parameter section